go-controller/pkg/config/config.go

```diff
@@ -537,6 +540,9 @@ type UserDefinedNetworksConfig struct {
 	// VRFTableBase will be the base to calculate the
 	// vrf table id for a user defined network, that's table=base+network-id
 	VRFTableBase uint `gcfg:"user-defined-network-vrf-table-base"`
+	//MasqueradeIPBase is an unexposed config with value 10
```

Comment:
Moving it to allocator, since we are not deriving this values from config as is done for other values 
like 
```golang
 // HostMasqConntrackZone is an unexposed config with the value of ConntrackZone+1
    HostMasqConntrackZone int                                                   
    // OVNMasqConntrackZone is an unexposed config with the value of ConntrackZone+2
    OVNMasqConntrackZone int                                                    
    // HostNodePortCTZone is an unexposed config with the value of ConntrackZone+3
    HostNodePortConntrackZone int                                               
    // ReassemblyConntrackZone is an unexposed config with the value of ConntrackZone+4
```